src/features/farming/cakeStall/CakeStall.tsx

```diff
@@ -65,17 +66,32 @@ export const CakeStall: React.FC = () => {
               }}
               onClick={openCakeModal}
             />
-            <img
-              src={goblin}
-              className="absolute"
-              style={{
-                width: `${GRID_WIDTH_PX}px`,
-                right: `${GRID_WIDTH_PX * 1.95}px`,
-                top: `${GRID_WIDTH_PX * 2.5}px`,
-                transform: "scaleX(-1)",
-              }}
-              onClick={openCakeModal}
-            />
+
+            {state.inventory["Chef Hat"] ? (
+              <img
+                src={chef}
+                className="absolute"
+                style={{
+                  width: `${GRID_WIDTH_PX * 1.2}px`,
+                  right: `${GRID_WIDTH_PX * 1.78}px`,
+                  top: `${GRID_WIDTH_PX * 1.8}px`,
+                  transform: "scaleX(-1)",
+                }}
+                onClick={openCakeModal}
+              />
```

Comment:
![Screen Shot 2022-08-29 at 5 30 12 pm](https://user-images.githubusercontent.com/41215134/187147737-f93a532a-7e92-47ef-a52f-22ca3824bcc2.png)

```suggestion
              <img
                src={chef}
                className="absolute"
                style={{
                  width: `${GRID_WIDTH_PX * 1.22222222222}px`,
                  right: `${GRID_WIDTH_PX * 1.71}px`,
                  top: `${GRID_WIDTH_PX * 2}px`,
                  transform: "scaleX(-1)",
                }}
                onClick={openCakeModal}
              />
```